docs/census_notebook_guidelines.md

```diff
@@ -13,7 +14,7 @@ These assets are user-facing and are automatically rendered to the doc-sites, th
 ### Title
 
 * It must use the highest-level markdown header `#`.
-* Unless needed, it should not contain "Cell Census".
+* Unless needed, it should not contain any direction mentions of "Census".
```

Comment:
Is there an alternative, or is the request to attempt no use of a "noun" to reference the collection?

Reply:
```suggestion
* Unless needed, it should not contain any direct mentions of "Census".
```
Is that what you meant?